deltacat/aws/redshift/model/manifest.py

```diff
@@ -29,13 +93,16 @@ def _build_manifest(
             manifest["entries"] = entries
         if author is not None:
             manifest["author"] = author
+        if entry_type is not None:
+            manifest["entry_type"] = entry_type.value
         return manifest
 
     @staticmethod
     def of(
         entries: ManifestEntryList,
         author: Optional[ManifestAuthor] = None,
         uuid: str = None,
+        entry_type: Optional[EntryType] = EntryType.get_default(),
```

Comment:
So only ManifestMeta has `entry_file_params`?

Reply:
`ManifestMeta` doesn't have `entry_file_params` as a parameter of its constructor. Only `ManifestEntry` does